src/main/java/salesforce/ui/PageTransporter.java

```diff
@@ -47,6 +50,12 @@ public CasesPage goToCases() {
      * @param pageName a String with the page name
      */
     public void goToPage(final String pageName) {
-        goToUrl(baseUrl.concat(SalesforceUrl.valueOf(pageName).getUrl()));
+        if ("classic".equals(salesforceVersion)) {
```

Comment:
As an opinion, maybe it would be more readable to use salesforceVersion.equals("classic")

Reply:
As I remember, Paolo once told us that it is better to use it in this way to avoid an error

Reply:
that's ok.